presto-spi/src/main/java/com/facebook/presto/spi/ColumnMetadata.java

```diff
@@ -151,4 +172,76 @@ public boolean equals(Object obj)
                 Objects.equals(this.extraInfo, other.extraInfo) &&
                 Objects.equals(this.hidden, other.hidden);
     }
+
```

Comment:
This isn't all of https://github.com/trinodb/trino/commit/6415b9aa9f5df0713389e2d4c87cfa71c4355cde
It makes sense as a separate commit more.

Reply:
Yeah, I'll commit separately next time. Thanks